api/main.py

```diff
@@ -86,7 +106,7 @@ def get_all_users():
     if not access:
         return jsonify({"users": None, "error": {"code": 401, "message": "Authentication failed"}})
     users = api.user_repository.get_all_users()
-    return jsonify({"users": [user.get_user_attributes() for user in users], "error": None})
+    return jsonify({"users": {[user.get_user_attributes() for user in users],['location']}, "error": None})
```

Comment:
You're making a set of tuples here. The first element of each tuple contains the user attributes, the second contains the (constant) list ['location']. This is useless.
You need to make sure that the get_user_attributes() method also exports the location field.